docker/airflow/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM apache/airflow:slim-latest-python3.7
```

Comment:
is python 3.7 a necessary version? Could we switch to a more recent version?

Reply:
sure thing. thanks for catching that. updated to 3.10